### ISSUE TITLE
When importing a framework in CASS, It is missing the first competency in the list after clicking on View in Editor from the import screen

### ISSUE THREAD

**Issue by sedula27** [cassproject/cass-editor#1365]
@miledivovic  
When a Competency Framework with 4 competencies is added to CASS via import, it says saving 5 objects and you can see all 4 competencies in the summary of the import but once you click on View in Editor option in the Import screen, We are missing the first competency in the list, I tested the export and it has 3 competencies instead of 4 that were uploaded into CASS. Refer the screen recording here.

https://github.com/user-attachments/assets/4b5e988b-b3ea-4eed-95e4-bf1d27bc5489
Export from CASS:https://cass.credentialengine.org/api/ceasn/5ab6d60d71571cee91a830c19d8fffab
Link to the Framework: https://apps.credentialengine.org/publisher/competencies/?frameworkctid=ce-71710f61-ef12-4b83-9a2e-1e38062d5466
CSV File used for upload: [Rasmussen_Competencies_updated - Record_CASS.csv](https://github.com/user-attachments/files/16460677/Rasmussen_Competencies_updated.-.Record_CASS.csv)
 

But once you delete the previously uploaded framework and re-import it, it has all 4 competencies

**Comment by winonadeb** [comment]
does this happen with other import files too?

**Comment by sedula27** [comment]
Yes, I did happen to other import files too, the workaround was to delete the first import and import again

**Comment by sedula27** [comment]
Here is a screen recording of the issue and workaround from Jeff
https://us02web.zoom.us/rec/share/BGMwU4Ppc0RnHYjfVZ5QzrEc7K0utv5mPTO8YS3pxdkAYfuxeG2Q_jJMLSZpBE4S.FlzTu101zEgN3OLl

**Comment by sedula27** [comment]
Not a consistent issue, it happened to frameworks uploaded from Rasmussen College

**Comment by sedula27** [comment]
Not able to reproduce it, leaving this issue open for future reference
Preserve the object created during the display

**Comment by sedula27** [comment]
Reopen this issue when/if it happens again
